virtcontainers/factory/template/template_amd64.go

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2018 HyperHQ Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// template implements base vm factory with vm templating.
+
+package template
+
+// templateDeviceStateSize denotes device state size when
+// mount tmpfs.
+// when bypass-shared-memory is not support like arm64,
+// creating template will occupy more space. That's why we
+// put it here.
+const templateDeviceStateSize = 8
```

Comment:
Nit: I'd personally probably create a `genericTemplateDeviceStateSize` variable and then for all the non-arm64 files:

```go
const templateDeviceStateSize = genericTemplateDeviceStateSize
```

That's the approach we've taken in `cli/kata-check.go` (`genericCPUFlagsTag`, *etc*).

Reply:
@jodh-intel that's really better.